target/linux/ath79/dts/ar7241_ubnt_xm.dtsi

```diff
@@ -9,6 +9,13 @@
 	compatible = "ubnt,xm", "qca,ar7241";
 	model = "Ubiquiti Networks XM (rev 1.0) board";
 
+	aliases {
+		led-boot = &link4;
```

Comment:
Since you're hijacking a led for a completely different purpose, you should limit the hijacking to the bare minimum. Please use the led only for boot and failsafe.

Reply:
I'm a bit skeptical about this. This will likely change the external behaviour known from ar71xx target where this LED is used as status LED, since there is no dedicated externally visible status LED on those boards.

This mapping did not conflict with RSSI function when I added "rssileds" package. rssileds daemon did simply override the status LED, and IMO should not conflict with status LED even when entering upgrade.

I will test the changed mapping later today and post an update. Thanks you for noticing.

Reply:
I tested the requested change, but I found, that expectedly, after the boot phase, LED4 turns off instead of staying solid. This is different behaviour from ar71xx port, and I consider this a part of user interface, which I'd like to keep stable.

Of course, the final decision is up to you, so I'm waiting for feedback :)

Reply:
drop it. we had the discussion multiple times before. don't hijack leds longer than necessary. the relay important signalisations are:

- still booting
- time to hit a button to get into failsafe
- we are in failsafe

all of the written above can be done by keeping only the boot and the failsafe led.

In ar71xx a lot of hacks were introduced. we are working hard on not copying them to ath79.

Reply:
OK, Understood. I'll post an update.